install.js

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-if (process.env.PUPPETEER_SKIP_CHROMIUM_DOWNLOAD) {
-  console.log('**INFO** Skipping Chromium download. "PUPPETEER_SKIP_CHROMIUM_DOWNLOAD" environment variable was found.');
+if (process.env.npm_config_puppeteer_skip_chromium_download || process.env.PUPPETEER_SKIP_CHROMIUM_DOWNLOAD) {
+  console.log('**INFO** Skipping Chromium download. "PUPPETEER_SKIP_CHROMIUM_DOWNLOAD" npm_config/environment variable was found.');
```

Comment:
let's console log explicitly why we're skipping download. It will save someone time to debug this behavior.

```js
if (process.env.PUPPETEER_SKIP_CHROMIUM_DOWNLOAD) {
  console.log('**INFO** Skipping Chromium download. "PUPPETEER_SKIP_CHROMIUM_DOWNLOAD" environment variable was found.');
  return;
}
if (process.env.npm_config_puppeteer_skip_chromium_download) {
  console.log('**INFO** Skipping Chromium download. "PUPPETEER_SKIP_CHROMIUM_DOWNLOAD" was set in npm config.');
  return;
}

```